### PR TITLE
[Bugfix] Remove activity indicator for chat when previous messages exhausted

### PR DESCRIPTION
**Pull request by jimchou-dev**
## Purpose

* Remove activity indicator for chat when previous messages exhausted
* Remove duplicate workspace reference to AzureCommunicationUIChat

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

**Review by JoshuaLai — DISMISSED** [review]
Tested and looks good

**Review by vhuseinova-msft — DISMISSED** [review]

**Review by jpeng-ms — DISMISSED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Service/Chat/ChatSDKWrapper.swift [hunk 1]
```diff
@@ -61,9 +61,11 @@ class ChatSDKWrapper: NSObject, ChatSDKWrapperProtocol {
                                 $0.toChatMessageInfoModel(
                                     localUserId: self.chatConfiguration.identifier.rawId)
                             })
+                        print("getInitialMessages success \(messages)")
```
Comment (palatter) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Service/Chat/ChatSDKWrapper.swift:64]: Left over debug?

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Service/Chat/ChatSDKWrapper.swift [hunk 2]
```diff
@@ -61,9 +61,11 @@ class ChatSDKWrapper: NSObject, ChatSDKWrapperProtocol {
...
                         continuation.resume(returning: messages?.reversed() ?? [])
                     case .failure(let error):
                         self.pagedCollection = nil
+                        print("getInitialMessages error \(error)")
```
Comment (palatter) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Service/Chat/ChatSDKWrapper.swift:68]: Left over debug?

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Redux/Reducer/RepositoryReducer.swift [hunk 3]
```diff
@@ -11,21 +11,29 @@ extension Reducer where State == RepositoryState,
         var lastUpdated = repositoryState.lastUpdatedTimestamp
         var hasFetchedInitialMessages = repositoryState.hasFetchedInitialMessages
         var hasFetchedPreviousMessages = repositoryState.hasFetchedPreviousMessages
+        var hasExhaustedPreviousMessages = repositoryState.hasExhaustedPreviousMessages
```
Comment (palatter) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Redux/Reducer/RepositoryReducer.swift:14]: Minor but can we call this hasFetchedAllMessages instead? Would match android and would be more clear that it's referring to fetching messages?

**Review by palatter — APPROVED** [review]
Looks good.

**Review by Leah-Xia-Microsoft — APPROVED** [review]

**Pull request merged**